rpc-server/src/main.rs

```diff
@@ -76,6 +76,8 @@ async fn main() -> anyhow::Result<()> {
     init_logging(false)?;
 
     let near_rpc_client = near_jsonrpc_client::JsonRpcClient::connect(opts.rpc_url.to_string());
+    // We want to set a custom referer to let NEAR JSON RPC nodes know that we are a read-rpc instance
+    let near_rpc_client = near_rpc_client.header(("referer", "read-rpc"))?; // TODO: make it configurable
```

Comment:
```suggestion
    let near_rpc_client = near_rpc_client.header(("Referer", "read-rpc"))?; // TODO: make it configurable
```

Reply:
I am not sure the `near-jsonrpc-client-rs` won't lowercase it under the hood though.